.github/workflows/cosmovisor-release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release Cosmovisor
+
+on:
+  push:
+    tags:
+      - "cosmovisor/v*.*.*"
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-go@v2.1.4
+        with:
+          go-version: 1.16
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/cosmovisor/}" >> $GITHUB_ENV
+      - name: Tag without prefix locally to avoid error in goreleaser
+        run: |-
+          git tag -d ${{ env.RELEASE_VERSION }} || echo "No such tag exists before"
+          git tag ${{ env.RELEASE_VERSION }} HEAD
```

Comment:
why do we need to make a local tag? What kind of error do you have in mind here (in the step name)?

Reply:
Why can't we use the full tag name and pass it directly to `GORELEASER_CURRENT_TAG` in the last step?

Reply:
> why do we need to make a local tag? What kind of error do you have in mind here (in the step name)?

Because goreleaser enforces semantic versioning and will error on non compliant tags. 
https://goreleaser.com/limitations/semver/
It should have semantic versioning to the HEAD, otherwise the pipeline will fail when trying to release to github.

> Why can't we use the full tag name and pass it directly to GORELEASER_CURRENT_TAG in the last step?
This is for the release page to associate with the correct tag with prefix `cosmovisor/v*.*.*` instead of the local tag `v*.*.*` in above step

Reply:
OK, could you add a comment about it in this yaml file please?

Reply:
added @robert-zaremba